lib/object.js

```diff
@@ -7,6 +7,11 @@ export default function inspectObject(object, options) {
     return '{}'
   }
   options.truncate -= 4
+  options.seen = options.seen || []
+  if (options.seen.indexOf(object) >= 0) {
+    return '[Circular]'
+  }
+  options.seen.push(object)
```

Comment:
This all looks good but maybe a `Set` is a better choice here?

Reply:
I thought that not all environment that loupe supported had a `Set`, so I used a simple array. Note that this is the implementation that was used in the `chaijs/chai` `inspect` function